src/net.h

```diff
@@ -966,8 +972,8 @@ class CNode
     std::vector<CAddress> vAddrToSend;
     std::unique_ptr<CRollingBloomFilter> m_addr_known{nullptr};
     bool fGetAddr{false};
-    std::chrono::microseconds m_next_addr_send GUARDED_BY(cs_sendProcessing){0};
-    std::chrono::microseconds m_next_local_addr_send GUARDED_BY(cs_sendProcessing){0};
+    std::chrono::microseconds m_next_addr_send{0};
+    std::chrono::microseconds m_next_local_addr_send{0};
```

Comment:
It might actually be better to turn these into atomics now that they're being accessed in both SendMessages and ProcessMessages. We know that those functions always run single threaded, but if someone exposes these values through an RPC then there' the potential of a data race.